canisters/src/vts/tests/agreement.rs

```diff
@@ -20,11 +20,20 @@ async fn test_create_agreement() {
             .await
             .unwrap();
     assert!(agreement_id_first > 0, "First agreement ID should be positive");
+}
+
+#[tokio::test]
+async fn test_sign_agreement() {
+    let (agent, canister_id) = init_agent().await;
+    let vh_customer = agent.get_principal().unwrap();
+    let name = "Test Agreement".to_string();
+    let daily_usage_fee = "100".to_string();
+    let gas_price = "10".to_string();
+
+    let agreement_id = create_agreement(&agent, &canister_id, &name, &vh_customer, &daily_usage_fee, &gas_price).await.unwrap();
 
-    // let result =
-    //     create_agreement(&agent, &canister_id, &name, &vh_customer, &daily_usage_fee, &gas_price)
-    //         .await;
-    // assert!(result.is_err(), "Should return error on duplicate agreement");
+    let result = sign_agreement(&agent,&canister_id, &agreement_id).await;
```

Comment:
Can you execute `make lint`, please? It looks like your source code is not formatted. We will do CI/CD later for it.